docs/dev/developer-guides/transactions/transactions.md

```diff
@@ -92,4 +102,23 @@ Instead of signing the RLP-encoded transaction, the user signs the following typ
 | factoryDeps            | `bytes32[]` |
 | paymasterInput         | `bytes`     |
 
-These fields are handled by our [SDK](../../../api/js/features.md).
\ No newline at end of file
+These fields are handled by our [SDK](../../../api/js/features.md).
+
+### Priority: `0xff`
+
+This is a zkSync Era specific transaction type related to a L1 -> L2 transaction.
```

Comment:
link to relevant page?

Reply:
We can add that since Etheruem, as L1, does not have a notion of "sending messages" to other layers, we had to make a specific tx type for it